Server/src/Models/Client.js

```diff
@@ -0,0 +1,14 @@
+
+const mongoose = require('mongoose');
+const business_schema = require('./Business.js');
+
+  // Schema
+  const client_schema = mongoose.Schema({
+    _id: { type: Number},
```

Comment:
Ref id's are automatically generated by mongo. Did you mean to use an internal ID?

Reply:
removed ids